heareval/tasks/util/luigi.py

```diff
@@ -105,6 +121,30 @@ def stage_number(self):
         else:
             raise ValueError(f"Unknown requires: {self.requires()}")
 
+    @property
+    def longname(self) -> str:
+        return f"{self.task_config['task_name']} {self.name}"
+
+    # This doesn't really log at all
```

Comment:
Huh weird, doesn't work well when their setup in objects as opposed to module level?

Reply:
yeah. can you set it up at the module level?